R/roxygen.R

```diff
@@ -27,14 +27,30 @@ roclet_output.roclet_dev <- function(x, results, base_path, ...) {
   blocks <- results
   dev_blocks <- Filter(block_has_dev, blocks)
   dev_topics <- sapply(dev_blocks, function(x) x$object$topic)
-  dev_files <- if (length(dev_topics)) file.path("^man", paste0(dev_topics, "\\.Rd$"))
+
+  if (length(dev_topics) == 0) {
+    return(invisible())
+  }
+
+  # .Rbuildignore
   ignored <- readLines(".Rbuildignore")
+  dev_files <- file.path("^man", paste0(nice_name(dev_topics), "\\.Rd$"))
```

Comment:
I noticed this because I have a topic called `.thing` whose Rd file is `dot-thing.Rd` :sweat_smile: